app/src/main/java/com/example/soccertask/ui/teamGenerator/TeamFragment.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.soccertask.ui.teamGenerator
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.soccertask.data.FixturesGenerator
+import com.example.soccertask.databinding.TeamGeneratorBinding
+import com.example.soccertask.data.TeamGenerator
+import com.example.soccertask.data.GenerateResults
+
+class TeamFragment: Fragment() {
+    private lateinit var binding: TeamGeneratorBinding
+    private lateinit var adapter: TeamGeneratorAdapter
+    private val teamsGenerator = TeamGenerator()
```

Comment:
Utilize ViewModel to deal with the Generators and Game data we want Fragment to be as "stupid" as possible